### ISSUE TITLE
Remove Unnecessary Images from assets/images 

### ISSUE THREAD

**Issue by slysnivy** [shoc71/Keep-On-Going#451]
Some of these images aren't being directly used in our game. There's hub_design, lvl_design, sketchs, and old_assets to name a few. We should move these images offshore into some drive or move them all into some special folder. With how it's scattered throughout the images folder, I'd have to actively filter out folders I don't want and ones I do and continually update this condition if new folders are added. 

If we fix this issue, then I don't have to have a check and update it for future image folders every time. Moving them to a special folder will easily resolve this issue but adds another location to keep track of for our game. If we put it in a special folder, then I don't need to continually update the filter, as it'll always be "do not open special_folder directory". Either approach is fine, but @shoc71 what do you think?

**Comment by shoc71** [comment]
I agree with that statement, that's why I made the old_assests folder where we dump any assets that we currently don't want to use. Just in-case we need it down the line or for another project (unlikely) there will always be a copy.

**Comment by slysnivy** [comment]
@shoc71 Ok, then how about we move all of the unnecessary folders under old_assets
